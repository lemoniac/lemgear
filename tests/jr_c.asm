	LD BC, $FFFF
	INC BC
	JR C, l
	
	LD E, 2
	
	HALT
	
l:
	LD E, 1
	
	HALT

