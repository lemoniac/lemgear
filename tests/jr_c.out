BC : 0000
DE : 0001
